qatrfm/utils/logger.py

```diff
@@ -11,57 +11,66 @@
 """ QaTrfm custom Logger Class
 
 It defines a specific format of the log messages.
-If LOG_COLORS=1 is exported, it will print the messages in different colors
-according to it's level.
 """
 
 import logging
-import os
 
 
 class QaTrfmLogger(logging.Logger):
 
-    def __init__(self, logger_name, level="DEBUG"):
+    colors = False
+
+    def __init__(self, logger_name):
         """Initialize QaTrfmLogger Class"""
-        self.colors = False
-        if ('LOG_COLORS' in os.environ):
-            self.colors = True
-            format = "\033[37;48mqatrfm.%(levelname)s: \033[0m%(message)s"
-        else:
-            self.colors = False
-            format = "qatrfm.%(levelname)s: %(message)s"
-        logging.basicConfig(level=logging.DEBUG, format=format)
-        return super(QaTrfmLogger, self).__init__(logger_name, level)
+        return super().__init__(logger_name)
+
+    @staticmethod
+    def colorize(msg, color):
+        COLORS_MAP = {
+                'blue': 34, 'green': 32, 'red': 31, 'yellow': 33,
+                'lightgrey': 37
+                }
+        if color in COLORS_MAP.keys():
+            return "\033[{}m{}\033[0m".format(COLORS_MAP[color], msg)
```

Comment:
```suggestion
            return "\033[1;{}m{}\033[0m".format(COLORS_MAP[color], msg)
```

I'd like to keep the colored messages bold. They don't look so nice (at least on my prompt) with light colors.
http://imagebin.suse.de/2706/img

@asmorodskyi  WDYT?

Reply:
I would use something like http://lnav.org/. So I don't see use case for this option

Reply:
never heard something about inav, sounds interesting. Are you using it somewhere, maybe you can show me.

Regarding bold, I don't have hard feelings, for me bold is good as well.

Reply:
nope I have never use it but I just wanted to show that idea of inserting such functionality inside qatrfm looks strange for me. In my understanding this problem could and should be solved outside. And actually comment from @jlausuch  with screenshots from **his** terminal is clearly showing why :) 
But I have no hard feelings about it after all can just avoid using this option :) Just discussion if it should be bold or not is looks strange :)

Reply:
I agree, that using colors has this difficulties.
So for me conclusion is, we will keep it for now. But looking for better way's when needed.

Reply:
ok